src/inc/til/rectangle.h

```diff
@@ -636,6 +632,32 @@ namespace til // Terminal Implementation Library. Also: "Today I Learned"
             return *this;
         }
 
+        // MUL will scale the entire rectangle up by the size factor
+        rectangle operator*(const size& size)
+        {
+            auto topLeft = _topLeft;
+            auto bottomRight = _bottomRight;
+            topLeft = topLeft * size;
+            bottomRight = bottomRight * size;

```

Comment:
stylistically, it looks meh, but it calls out the actions explicitly so i'm fine w/

Reply:
Eh, I condensed it a little.